bin/mock-outcomes

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+from sentry.runner import configure
+
+configure()
+
+from datetime import datetime, timedelta
+
+import requests
+from django.conf import settings
+
+from sentry.constants import DataCategory
+from sentry.utils import json
+from sentry.utils.outcomes import Outcome
+
+# import random
+
+
+def store_outcomes(outcome, num_times=1):
```

Comment:
What are outcomes? Could you please add a docstring explaining why we need to store the data in Snuba?

Reply:
So this is just adding mock data to the outcomes dataset. We pull from that dataset in snuba, so if the dataset was empty the weekly emails won't be sent